woodwork/statistics_utils.py

```diff
@@ -300,6 +300,30 @@ def _get_mutual_information_dict(dataframe, num_bins=10, nrows=None, include_ind
     return mutual_info
 
 
+def _get_valid_mi_columns(dataframe, include_index=False):
+    """Returns a list of column names a list of columns from the DataFrame with valid
+        Logical Types that support mutual information.
```

Comment:
I think there's a typo here.
```suggestion
    """Returns a list of columns from the DataFrame with valid
        Logical Types that support mutual information.
```